src/modules/map/components/openlayers/OpenLayersExternalWMTSLayer.vue

```diff
@@ -29,11 +30,39 @@ const { externalWmtsLayerConfig, parentLayerOpacity, zIndex } = toRefs(props)
 // mapping relevant store values
 const store = useStore()
 const projection = computed(() => store.state.position.projection)
+const previewYear = computed(() => store.state.layers.previewYear)
+const isTimeSliderActive = computed(() => store.state.ui.isTimeSliderActive)
 
 // extracting useful info from what we've linked so far
 const layerId = computed(() => externalWmtsLayerConfig.value.id)
 const opacity = computed(() => parentLayerOpacity.value ?? externalWmtsLayerConfig.value.opacity)
-const options = computed(() => externalWmtsLayerConfig.value.options)
+const options = computed(() => {
+    const options = { ...externalWmtsLayerConfig.value.options }
+    options.dimensions = { ...options.dimensions }
```

Comment:
Might make sense that we add some cloneDeep functions from Lodash or other library for such cases